src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in

```diff
@@ -7,7 +7,6 @@ Name: @PROJECT_NAME@-libapi
 Description: @PROJECT_DESCRIPTION@
 URL: @PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
-Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@
```

Comment:
Similarly, these need to remain. Otherwise, projects using `pkg-config` to consume the SDK will fail to link.

Reply:
I just read your comment that said "remove Requires line -- it adds like a minute of CPU-bound processing inside the consumer's cmake config". I don't see how that could be true. These files aren't consumed by cmake, only by `pkg-config`.

Reply:
example run with `Requires` in pkg-config (last line has time):
```
# cmake .. -G Ninja
-- The CXX compiler identification is GNU 12.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.8.1") 
-- Checking for one of the modules 'viam-cpp-sdk-libviamsdk'
-- Checking for one of the modules 'viam-cpp-sdk-libviamapi'
-- Configuring done (31.1s)
```

Reply:
Ah, I see. But why are you using `pkg-config` to obtain the SDK rather than using `find_pacakge`, which will use the cmake infra that the SDK provides?